lib/db.js

```diff
@@ -82,6 +84,12 @@ class LimitDBRedis extends EventEmitter {
     this.redis.on('node error', (err, node) => {
       this.emit('node error', err, node);
     });
+
+    this.cache = config.cache || new TTLCache({
+      max: 1000,
+      ttl: 1000,
+      checkAgeOnGet: true
```

Comment:
Does this cause `get` returns _undefined_ if the TTL has met?

Reply:
Correct.